coreapi/response.go

```diff
@@ -230,3 +231,33 @@ type ScheduleResponse struct {
 	PreviousExecutionAt string `json:"previous_execution_at"`
 	NextExecutionAt     string `json:"next_execution_at"`
 }
+
+type PaymentAccountResponse struct {
+	StatusCode             string   `json:"status_code"`
+	PaymentType            string   `json:"payment_type"`
+	AccountId              string   `json:"account_id"`
+	AccountStatus          string   `json:"account_status"`
+	ChannelResponseCode    string   `json:"channel_response_code"`
+	ChannelResponseMessage string   `json:"channel_response_message"`
+	Action                 Action   `json:"action"`
+	Metadata               Metadata `json:"metadata"`
+	StatusMessage          string   `json:"status_message"`
+	ID                     string   `json:"id"`
+}
+
+type Metadata struct {
```

Comment:
Better name the type `Metadata` as `PaymentAccountMetadataDetails`.

So the struct field become:
```
Metadata               PaymentAccountMetadataDetails `json:"metadata"`
```

Reply:
done